protected/humhub/modules/admin/views/group/edit.php

```diff
@@ -47,13 +52,13 @@
     <?= $form->field($group, 'show_at_directory')->checkbox(); ?>
     <?= $form->field($group, 'sort_order')->widget(SortOrderField::class) ?>
 
-    <?= CHtml::submitButton(Yii::t('AdminModule.user', 'Save'), ['class' => 'btn btn-primary', 'data-ui-loader' => ""]); ?>
+    <?= Html::submitButton(Yii::t('AdminModule.base', 'Save'), ['class' => 'btn btn-primary', 'data-ui-loader' => ""]); ?>
```

Comment:
Just `base` is suffient here:

Example
```php 
<?= Html::submitButton(Yii::t('base', 'Save')
```

Reply:
or

```
<?= Button::save()->submit() ?>
```

Reply:
Done!